frontend/apps/reader/modules/readergesture.lua

```diff
@@ -1631,4 +1639,14 @@ function ReaderGesture:onToggleReadingOrder()
     return true
 end
 
+function ReaderGesture:getCurrentPage()
```

Comment:
We don't already have something like this? And are we sure this is where it should go?

Reply:
not sure, if we do please let me know it would be helpfull all over the place :)
right now we have a bunch of 
```lua
    if self.document.info.has_pages then
        page = self.ui.paging.current_page
    else
        page = self.document:getCurrentPage()
    end
```
all over the codebase. so i assumed there wasnt one yet.

In general i think this should be fixed everywhere

Reply:
I think something like `Document:getCurrentPage` would probably make the most sense.

Reply:
a qiuck `ag` shows
https://github.com/koreader/koreader/blob/8d8b8cd0f79455ac4e4bc1dc25524a98a24cdeea/frontend/apps/reader/modules/readergesture.lua#L1568
https://github.com/koreader/koreader/blob/8d8b8cd0f79455ac4e4bc1dc25524a98a24cdeea/frontend/apps/reader/skimtowidget.lua#L66
https://github.com/koreader/koreader/blob/8d8b8cd0f79455ac4e4bc1dc25524a98a24cdeea/frontend/apps/reader/skimtowidget.lua#L424
https://github.com/koreader/koreader/blob/8d8b8cd0f79455ac4e4bc1dc25524a98a24cdeea/plugins/statistics.koplugin/main.lua#L680